internal/client.go

```diff
@@ -107,6 +107,13 @@ type ClientOptions struct {
 	Credentials       primitive.Credentials
 }
 
+func NewClientOptions() ClientOptions{
```

Comment:
why not use `func DefaultClientOptions() ClientOptions {...}`, refer: L87

Reply:
done